jvm/build.gradle

```diff
@@ -4,18 +4,12 @@
 
 
 buildscript {
-    ext.kotlin_version = '1.1.0'
-
     repositories {
         mavenCentral()
     }
-
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
 }
 
-apply plugin: 'kotlin'
```

Comment:
Can you please explain this to me? We didn't have "Java" plugin earlier but we did have a mix of Kotlin and Java.